openapi/components/schemas/QuickBooksOnlineExternalServiceSettings.yaml

```
@@ -0,0 +1,123 @@
+
+type: object
+description: QuickBooks Online settings.
+properties:
+  customer:
+    type: object
+    title: QuickBooks Online customer external service settings
+    properties:
+      sync:
+        type: string
+        description: Specifies how to sync customers.
+        default: manually
+        enum:
+          - manually
+          - when-used
+          - always
+        x-enumDescriptions:
+          manually: Only via a manual sync.
+          when-used: Whenever related entity is synced (transaction, invoice).
+          always: Create and update customers automatically.
```

nitpick: Good to give more detail here

(Also, we should avoid [Latin](https://product.rebilly.dev/writing-style-guide/cheat-sheet/#do-not-use) and abbreviations)  

```suggestion
          manually: Customer data is only synchronized when it is manually executed.
          when-used: Customer data is synchronized when related entities, such as transactions or invoices are changes or updated.
          always: Customer data is synchronized automatically as data is created or updated.
```

---

Fixed a typo here
```suggestion
          manually: Customer data is only synchronized when it is manually executed.
          when-used: Customer data is synchronized when related entities, such as transactions or invoices are created or updated.
          always: Customer data is synchronized automatically as data is created or updated.
```

---

Thanks for spotting this!